test/e2e/prepare.go

```diff
@@ -43,6 +45,12 @@ func waitForDaemonSets(t *testing.T, kubeclient kubernetes.Interface, namespace
 			return true, errors.Wrapf(err, "failed retrieving daemon sets for namespace %s", namespace)
 		}
 		for _, daemonset := range daemonsets.Items {
+			// we don't have the app=kubernetes-nmstate labels at daemonsets so we
+			// filter by name
+			if !strings.Contains(daemonset.Name, "nmstate") {
```

Comment:
It is not great, but we can just check the app label in daemonset.Spec.Template. (We should add proper labels on master)

Reply:
I would keep this like this, browsing Spec.Template add more complicated code, let's add app label at master and we fix this.

https://github.com/nmstate/kubernetes-nmstate/issues/476